sdk/search/azure-search-documents/src/test/resources/session-records/batchTakesAllNonDuplicateKeys.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
valid/expected output?

Reply:
This is a non-live test as getting the service to return a 413/503 is very difficult so mocking is being used